wagon-providers/wagon-http-shared/src/main/java/org/apache/maven/wagon/shared/http/StandardServiceUnavailableRetryStrategy.java

```diff
@@ -28,17 +28,13 @@
 import org.apache.http.util.Args;
 
 /**
- * An implementation of the {@link SimpleServiceUnavailableRetryStrategy} interface.
+ * An implementation of the {@link ServiceUnavailableRetryStrategy} interface.
  * that retries {@code 408} (Request Timeout), {@code 420} (Method Failure), {@code 429} (Too Many Requests),
  * and {@code 500} (Server side error) responses for a fixed number of times at a fixed interval.
  */
 @Contract( threading = ThreadingBehavior.IMMUTABLE )
-public class SimpleServiceUnavailableRetryStrategy implements ServiceUnavailableRetryStrategy
+public class StandardServiceUnavailableRetryStrategy implements ServiceUnavailableRetryStrategy
```

Comment:
I am not convinced that this should be really called standard because of >= 500 and there seems to many intepretations of 420 and all of them are deprecated or not present: https://www.whoishostingthis.com/resources/http-status-codes/#method-failure-400.

I am fine with 408, 421, 503 and 504. 

Please tell me why all 5xy are necessary.